src/components/Contact/ContactSponsorUs/ContactSponsorUs.js

```diff
@@ -1,20 +1,42 @@
+import './ContactSponsorUs.css';
+
 import React, { Component } from 'react';
+
+import Col from 'react-bootstrap/Col'
 import ContactSponsorPackages from './ContactSponsorPackages/ContactSponsorPackages.js';
-import './ContactSponsorUs.css';
+import Row from 'react-bootstrap/Row'
 
 class ContactSponsorUs extends Component {
   constructor(props) {
     super(props);
 
     this.sponsorsData = props.sponsorsData;
-
+  
     this.sponsorUsCallback = this.sponsorUsCallback.bind(this);
+    this.translate = this.translate.bind(this);
+
+    this.state = {
+      /** @type { string } */
+      language: 'en',
+      /** @type { string } */
+      translateLabel: 'Translate to spanish',
+      /** @type { string } */
+      sponsorButtonLabel: 'Sponsor us!'
+    };
   }
 
   sponsorUsCallback() {
     window.open(this.sponsorsData.url_contact, '_blank');
   }
 
+  translate() {
```

Comment:
Good proposal, but I think the logic is inverted. The idea is that, if the sponsor button states `Sponsor Us!` then the translate button will state `Traducir a español`. So if one button is in Spanish then the other is in English to clarify the translation option. Does that make sense?